cmd/create.go

```diff
@@ -8,6 +8,8 @@ import (
 	"text/tabwriter"
 	"text/template"
 
+	flag "github.com/spf13/pflag"
+
```

Comment:
Just a nit, any reason for this empty line?

Reply:
Nope, copy&paste from `goimports`, I updated it as expected and `goimports` does not complain. Strange.